Makefile.common

```diff
@@ -175,6 +176,9 @@ ifeq ($(SYSTEM_ZLIB), 1)
 	CFLAGS += $(shell pkg-config --cflags zlib)
 	LDFLAGS += $(shell pkg-config --libs zlib)
 else
+ifneq (,$(findstring unix,$(platform)))
+	PLATCFLAGS += -DHAVE_UNISTD_H
```

Comment:
Why is this required?

Reply:
Without it `custom/dependencies/libzlib/zconf.h` doesn't include unistd.h and compilation fails on unknown functions read(), close(), write().

I don't insist on this solution, but something is needed to remedy this issue.

Reply:
Hm how didnt this blow before? Did it link implicitly? Not sure if large file support would be more appropriate.

Reply:
No idea. It's not something I have tested previous versions against.

Reply:
I just compiled this core on a Debian Trixie, which is a testing release but not as experimental as Fedora Rawhide is. The GCC version here is 13 as opposed to GCC 14 in Rawhide. The implicit declarations are reported by GCC 13 as well, but only as warnings. Most likely this warning has been promoted to an error on somewhere the way to GCC 14.1.

Reply:
https://gcc.gnu.org/gcc-14/porting_to.html meh